gymnasium/spaces/utils.py

```diff
@@ -137,19 +157,21 @@ def flatten(space: Space[T], x: T) -> FlatType:
 
 @flatten.register(Box)
 @flatten.register(MultiBinary)
-def _flatten_box_multibinary(space, x) -> np.ndarray:
+def _flatten_box_multibinary(space: Box | MultiBinary, x: NDArray[Any]) -> NDArray[Any]:
     return np.asarray(x, dtype=space.dtype).flatten()
 
 
 @flatten.register(Discrete)
-def _flatten_discrete(space, x) -> np.ndarray:
+def _flatten_discrete(space: Discrete, x: int) -> NDArray[np.int64]:
```

Comment:
`NDArray[np.integer]`? Doesn't have to be `int64` afaik

Reply:
`Discrete.dtype` is always `np.int64` so I think this is correct